programs/survival/revive_coral

```diff
@@ -0,0 +1,14 @@
+//allows player to revive dead corals with water bottles
+__on_player_right_clicks_block (player, item_tuple,hand, block,face,hitvec) -> (
+	l(item, count, nbt) = item_tuple || l('None', 0, null);
+	position=pos(block);
+	types=l('brain','bubble','fire','horn','tube');
+	block=block-'dead_'-'_coral_block';
+	if(item=='potion'&& types ~ block != null && nbt=='{Potion:"minecraft:water"}',
```

Comment:
that would work if that's all what's in the nbt tag. You could make it more robust by using `nbt:'Potion' == 'minecraft:water'`

Reply:
thx I will use